doc/developers/develop.rst

```diff
@@ -520,6 +520,9 @@ poor_score (default=``False``)
     ``make_blobs(n_samples=300, random_state=0)``. These datasets and values
     are based on current estimators in sklearn and might be replaced by
     something more systematic.
+    
+preserves_32bit_dtype (default=``False``)
+    whether the transformer preservers ``np.float32`` if given as input.
```

Comment:
```suggestion
    whether an estimator preserves `np.float32` if given as input.
```

Reply:
do we want only this test for the transformer? It should be added that this does not apply to clustering method.

Reply:
Well I think testing transformers is a starts, I'm not sure what other estimators this could apply to.